pkg/factory/risingwave_object_factory.go

```diff
@@ -1564,11 +1573,26 @@ func (f *RisingWaveObjectFactory) portsForStandaloneContainer() []corev1.Contain
 
 func (f *RisingWaveObjectFactory) setupStandaloneContainer(container *corev1.Container) {
 	container.Name = "standalone"
-	container.Command = []string{
-		risingwaveExecutablePath,
-		"standalone",
+
+	var imageVersion = utils.GetVersionFromImage(f.risingwave.Spec.Image)
+	if strings.HasPrefix(imageVersion, "v1.3") ||
+		strings.HasPrefix(imageVersion, "v1.4") ||
+		strings.HasPrefix(imageVersion, "v1.5") ||
+		strings.HasPrefix(imageVersion, "v1.6") ||
+		strings.HasPrefix(imageVersion, "v1.7") {
```

Comment:
Prevent from false positive 

```suggestion
        // `standalone` subcommand is only supported in versions 1.3 - 1.7, and was deprecated in 1.8.
	if strings.HasPrefix(imageVersion, "v1.3.") ||
		strings.HasPrefix(imageVersion, "v1.4.") ||
		strings.HasPrefix(imageVersion, "v1.5.") ||
		strings.HasPrefix(imageVersion, "v1.6.") ||
		strings.HasPrefix(imageVersion, "v1.7.") {
```